setup.cfg

```
@@ -23,7 +23,7 @@ ignore =
     W504
 
 [tool:pytest]
-addopts = --cov=./ -W ignore::DeprecationWarning
+addopts = --cov=otio_aaf_adapter -W ignore::DeprecationWarning
```

```suggestion
addopts = --cov=otio_aaf_adapter
```
It is generally better if CI logs (or even fails, `-W error::DeprecationWarning`) when `DeprecationWarning`s are encountered.

---

It is like that in the template repo, but I agree

---

We could try turning that off to see if it passes. I suspect that since we're running against such a wide range of Python versions that some deprecations are unavoidable...

---

I've already turned it off ignore and all the tests still pass.  I didn't try `-W error::DeprecationWarning` though, I thought that would be overkill for now.